tests/unit/verifiers/test_base.py

```diff
@@ -94,6 +94,16 @@ def test_base_verifier_warn_on_unchecked_units(mocker):
 
     assert expected_partial_result not in result.partials
 
+    # Test run excluding missing unit explicitly
+    log_warning.reset_mock()
```

Comment:
This is not used anywhere. Could you drop it?

Reply:
Good catch. This is probably a remnant from before the `Result` reporting was reworked. The whole `log_warning` mock can be removed.